articles/connections/social/bitbucket.md

```diff
@@ -14,85 +12,49 @@ useCase:
     - customize-connections
     - add-idp
 ---
-# Connect your app to Bitbucket
+# Connect your App to Bitbucket
 
 To connect your Auth0 app to Bitbucket, you will need to generate a *Key* and *Secret* in a Bitbucket app, copy these keys into your Auth0 settings, and enable the connection.
 
 ::: note
   This connection will only work with Lock version 9.2 or higher.
 :::
 
-## 1. Login to Bitbucket
+1. Login to [Bitbucket](https://bitbucket.org/). Click on your account icon in the upper right and select **Bitbucket settings**. Select **OAuth**  in the left nav.
 
-Login to [Bitbucket](https://bitbucket.org/).
-
-Click on your account icon in the upper right and select **Bitbucket settings**:
-
-![](/media/articles/connections/social/bitbucket/bitbucket-01.png)
-
-Select **OAuth**  in the left nav:
-
-![](/media/articles/connections/social/bitbucket/bitbucket-02.png)
-
-## 2. Create your app
-
-Click **Add consumer**.
-
-Provide a name for your app.
-
-In the Callback URL field, enter the folowing:
+2. To create your app, click **Add consumer**. Provide a name for your app. In the **Callback URL** field, enter the folowing:
 
 `https://${account.namespace}/login/callback`
 
-Select the Permissions you want to enable for this connection. At the very least you will need to select the `Account:Email` and `Account:Read` permissions.
-
-Click **Save**.
+3. Select the Permissions you want to enable for this connection. At the very least you will need to select the `Account:Email` and `Account:Read` permissions.
 
-![](/media/articles/connections/social/bitbucket/bitbucket-03.png)
+4. Click **Save**.
 
-## 3. Get your *Key* and *Secret*
+5. To get your *key* and *secret*, on the page that follows, click the name of your app under **OAuth consumers** to reveal your keys.
 
-On the page that follows, click the name of your app under **OAuth consumers** to reveal your keys:
-
-![](/media/articles/connections/social/bitbucket/bitbucket-04.png)
-
-## 4. Copy your *Client Id* and *Client Secret* into Auth0
-
-In a separate window, login to your [Auth0 Dashboard](${manage_url}) and select **Connections > Social** in the left nav.
-
-Select **Bitbucket**:
+6. Copy your *Client Id* and *Client Secret* into Auth0. In a separate window, log in to your [Auth0 Dashboard](${manage_url}) and select **Connections > Social** in the left nav. Select **Bitbucket**:
 
 ![](/media/articles/connections/social/bitbucket/bitbucket-05.png)
 
-Copy the `Key` and `Secret` from the **OAuth integrated applications** page on Bitbucket into the fields on this page on Auth0.
-
-Select the **Permissions** you want to enable.
-
-Click **SAVE**.
+7. Copy the `Key` and `Secret` from the **OAuth integrated applications** page on Bitbucket into the fields on this page on Auth0. Select the **Permissions** you want to enable. Click **SAVE**.
```

Comment:
You changed the case of key and secret above, same here?

Reply:
Good catch. I changed it back so they all match (title case).

Reply:
I'm not replacing the screenshots. There were superfluous and didn't add much at all. I kept the relevant ones (that hadn't changed). This way, if Bitbucket modifies their UI in the future, we won't have to chase updates.